### PR TITLE
Fixes #41

### LINKED ISSUE #41: Parenthesis Checker
I'd like to contribute to the Parenthesis Checker Question
https://github.com/DSC-Banasthali-Vidyapith/Code-Crossroad/pull/34

### PR DESCRIPTION
**Pull request by An3sha**
Parenthesis Checker Solution in Cpp

### CODE REVIEW

File: Stacks/Parenthesis_Checker.cpp [hunk 1]
```diff
@@ -0,0 +1,93 @@
+ // Anisha Singh
```
Comment (sakshigoel19) [inline Stacks/Parenthesis_Checker.cpp:1]: Remove your initials in the beginning. Instead add the description of your algorithm using multi-line comments at the starting of the code.
```suggestion
 // Anisha Singh
```

File: Stacks/Parenthesis_Checker.cpp [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+bool check_parenthesis(string str) 			 //Boolean Function to check the given expression 
+{
+	stack <char> s;
+
+	for(int i=0; i<str.length(); i++)
+	{
+		if(str[i]=='(' || str[i]=='[' || str[i]=='{' )
+		{
+			s.push(str[i]);    					//Push the element into stack
```
Comment (sakshigoel19) [inline Stacks/Parenthesis_Checker.cpp:18]: Add comments above the line of code.

File: Stacks/Parenthesis_Checker.cpp [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+   // Output:  True
+
+   // Input : [)]
+   // Output:  False
+ 
+ 	// Time Complexity : O(n)
+ 	// Space Complexity : O(n)
+	
+
+
```
Comment (sakshigoel19) [inline Stacks/Parenthesis_Checker.cpp:93]: Use multi-line comments at the end and remove unnecessary blank lines.

File: Stacks/Parenthesis_Checker.cpp (hunk 1 above)
Comment (An3sha) [reply Stacks/Parenthesis_Checker.cpp]: I resolved the issues. Thank you for guiding.

**Review by Vanshikagarg17 — APPROVED** [review]

**Pull request merged**